roles/update/tasks/create_local_openstackclient.yml

```diff
@@ -80,7 +16,22 @@
 
 - name: Set the openstackclient image fact
   ansible.builtin.set_fact:
-    openstackclient_image: "{{ _cifmw_update_openstackclient_pod.resources[0].spec.containers[0].image }}"
+    openstackclient_image: "{{ _cifmw_update_openstackclient_pod | community.general.json_query('resources[0].spec.containers[0].image') | default('') }}"
```

Comment:
nit: suggest to add one more task with `fail` that will verify if the openstackclient_image is not empty

Reply:
There is Fail task if pod is not found before. If pod is found, then image should be there too